### ISSUE TITLE
Board not working and cannot sort the library

### ISSUE THREAD

**Issue by Lamp-Post** [Stremio/stremio-bugs#24]
The Board shows no notifications or recommendations, while I have added multiple series and films.
And I cannot sort the order from A-Z and/or Watched in the Library, both for Series and Films

**Comment by Ivshti** [comment]
Give it some time to update. Considering how the current system works, if you don't see anything in the Board after 10-20 minutes, THEN something is wrong.

**Comment by Lamp-Post** [comment]
It has already been more than a day. So unfortunately there is a problem

**Comment by Ivshti** [comment]
Aer you a guest user or signed up?

**Comment by Lamp-Post** [comment]
I signed up, I have send you an email

**Comment by Ivshti** [comment]
This was a visual glitch @Lamp-Post 
The issue was that when you're using Guest Login, it should show "Please create an account to use the Board" instead of "please wait a few minutes"

The Board doesn't work for Guest mode, because the Library doesn't sync in guest mode.

**Comment by Lamp-Post** [comment]
But I am not using Guest Login...

**Comment by Draxler** [comment]
Why there is no Board available for guest account users anymore ? 
In previous Stremio version i had working Board section.

**Comment by Ivshti** [comment]
@Draxler because the Board is processed on the server side, and this requires the Library to be synced to the servers.

There are two reasons to not do this anymore:
1. Syncing your library is something that isn't appropriate for Guest users, since 'guest' implies anonymity 
2. Ease the load off the servers for users who do not want Library / Board; whoever wants them should register, it's not that hard

**Comment by Lamp-Post** [comment]
I do not know how, but my Board is working! So thank you. But I still have the problem of not being able to sort my Library

**Comment by Ivshti** [comment]
@Lamp-Post yeah, we'll fix this with the next release. Today or tomorrow.

**Comment by asolvang** [comment]
Hey, I now have this problem on a Windows 10 pc, started during the last weekend sometime. I have tried to log out and in, still not working. New episodes are showing up in the Library, but not on the board (neither are movies). Only a "blank" screen with the Stremio background color. Don`t think I have updated Stremio or Windows 10 lately. Tried reinstalling and currently have version 3.6.5. Let me know if you want/need more system info.
